main.tex

```diff
@@ -342,84 +321,133 @@ \subsection{Best response memory-one strategies}\label{section:best_response_mem
+\end{equation}
+
+Thus, SSE is defined as how far a strategy is from behaving as a ZD and thus a
+high SSE implies a non extortionate behaviour. The SSE method has been applied
+to the data set. The distributions of SSE for the best response in tournaments
+with and without self interactions are given in
+Figure~\ref{fig:sse_distributions}. Moreover, a statistics summary of the SSE
+distributions is given in Table~\ref{table:sserror_stats}.
+
+\begin{table}[!htbp]
+    \begin{center}
+    \resizebox{.8\columnwidth}{!}{%
+    \begin{tabular}{lrrrrrrrrrrr}
+        \toprule
+        & mean & std  & 5\% & 50\% &  95\% & max & median & skew & kurt\\
+        \midrule
+    \textbf{Tournament without self interactions} & 0.34  & 0.40  & 0.028  & 0.17  &
+    1.05  & 2.47  & 0.17  & 1.87 & 3.60 \\
+    \textbf{Tournament with self interactions} & 0.17 & 0.23 & 0.01 &
+    0.12 & 0.67 & 1.53 & 0.12 & 3.42 & 1.92 \\
+        \bottomrule
+    \end{tabular}}
+    \end{center}
+    \caption{SSE of best response memory-one when \(N=2\)}\label{table:sserror_stats}
+\end{table}
+
+\begin{figure}[!htbp]
+    \begin{subfigure}{0.47\textwidth}
+        \begin{center}
+            \includegraphics[width=\linewidth]{img/best_respones_sserror.pdf}
+        \end{center}
+        \caption{SEE distribution for best response in tournaments without self interactions.}
+    \end{subfigure}\hfill
+    \begin{subfigure}{0.47\textwidth}
+        \begin{center}
+            \includegraphics[width=\linewidth]{img/evo_sserror.pdf}
+        \end{center}
+        \caption{SEE distribution for best response in tournaments with self interactions.}
+    \end{subfigure}
+    \caption{SEE distributions for best response in tournaments without and with self interactions.}\label{fig:sse_distributions}
+\end{figure}
+
+For the best response in tournaments that do not include self interactions the
+distribution of SSE is skewed to the left, indicating that the best response
+does exhibit ZDs behaviour and so could be extortionate, however, the best
+response is not uniformly a ZDs. A positive measure of skewness and kurtosis,
+and a mean of 0.34 indicate a heavy tail to the right. Therefore, in several
+cases the strategy is not trying to extort its opponents. More specifically,
+the best response memory-one with a theory of mind is being more adaptable than a ZDs.
+In~\cite{Hilbe2018} a similar behaviour is refereed to as the \textit{partners
+strategy}. Partners
+strategy aims to share the payoff for mutual cooperation, but it is ready
+to fight back when being exploited. In~\cite{Hilbe2018} they showed that partner strategies are
+selected in evolutionary settings when populations are large and relationships
+stable and they allow for the evolution of cooperation. A best response memory-one strategy
+in evolutionary settings has not been incorporated in the study of this manuscript, however,
+the best response memory-one strategy with self interactions can used to approximate
```

Comment:
```suggestion
the best response memory-one strategy with self interactions can used to approximate % TODO Vince to work on this a bit
```

Essentially we're saying:

- Best response mem1: adapt they are sometimes ZD and sometimes not;
- There is a paper by Hilbe that described a strategy designed to do the same thing and was evolutionary stable.
- That's cool.